src/parser/AbbrMarker.ts

```diff
@@ -7,7 +7,7 @@ const isCapitalized = (text: string) => {
     if (!text || text.length === 0) {
         return false;
     }
-    return /^[A-Z]/.test(text);
+    return /^\p{Lu}/u.test(text);
```

Comment:
📝 Node.js 10+ has supported [Unicode character class escape: \p{...}, \P{...} - JavaScript | MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape).

Reply:
I prefer to use `/^\p{Uppercase_Letter}/`　because I needed to find out what Lu meant.

https://www.regular-expressions.info/unicode.html#category

Reply:
Changed.